plugins/README.md

```diff
@@ -37,14 +39,37 @@ plugins.
 - [``@blockly/theme-modern``](theme-modern/): A Blockly modern theme.
 - [``@blockly/theme-dark``](theme-dark/): A Blockly dark theme.
 
-### Other
-
+### Block Extension
+
+- [``@blockly/block-extension-tooltip``](block-extension-tooltip/): A Blockly
+block extension that adds support for custom tooltip rendering.
+
+### Workspace
```

Comment:
I've created a workspace section, trying to follow the ctageogy of plugin types described in our [documentation](https://developers.google.com/blockly/guides/plugins/naming). I'm not sure whether I'm correct in how I categorized them.